xkill@eson57/xkill@eson57.nemo_action.in

```diff
@@ -0,0 +1,8 @@
+[Nemo Action]
+_Name=XKill
+Exec=<xkill@eson57/xkill@eson57.sh>
+Icon-Name=application-exit-symbolic
+Selection=none
+Extensions=any;
+Conditions=desktop;
```

Comment:
The conditions are tough because `xkill` technically isn't supported in Wayland but it does work for XWayland applications in a Wayland session. Maybe also include a one-time Zenity window in the helper script, conditional on a Wayland session being detected?

Reply:
Sure, but I'll need help for this. You know I'm not really a dev, just slowly learning because I think it's fun. ;-)

Reply:
Which part(s) do you need help with? You can do a conditional in your helper script to check if it's the Wayland session and then do the same hidden file that you have already done. That Zenity window would set the proper expectations for the user in the Wayland session.

Reply:
Well testing shows that it's already working as is, in a Wayland session. So if I've not misunderstood something, I will leave it as is.

Reply:
As you documented, it will work for Xwayland applications but it will not work for Wayland-native applications. Firefox has a flag you can set to make it Wayland-native if you want to test it. Chromium-based browsers also have similar settings.